Assets/Scripts/Models/Buildable/Furniture.cs

```diff
@@ -691,6 +697,11 @@ public void ReadXmlPrototype(XmlReader readerParent)
                     reader.Read();
                     Height = reader.ReadContentAsInt();
                     break;
+                case "Health":
+                    reader.Read();
+                    health.InitialHealth(reader.ReadContentAsFloat());
+                    health.MaxHealth = reader.ReadContentAsFloat();
```

Comment:
Wouldn't this be better as `health = new Health(reader.ReadContentAsFloat());` and then wouldn't it make `InitialHealth(float value)` redundant?